packages/paste-core/components/progress-steps/src/ProgressStepContent.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, type BoxProps } from "@twilio-paste/box";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+export interface ProgressStepContentProps extends HTMLPasteProps<"div"> {
+  children?: React.ReactNode;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default 'PROGRESS_STEP_CONTENT'
+   * @type {BoxProps['element']}
+   * @memberof ProgressStepContentProps
+   */
+  element?: BoxProps["element"];
+}
+
+export const ProgressStepContent = React.forwardRef<HTMLDivElement, ProgressStepContentProps>(
+  ({ element = "PROGRESS_STEP_CONTENT", ...props }, ref) => {
+    return (
+      <Box element={element} color="colorText" ref={ref} paddingY="space40">
```

Comment:
hmm good question. I say yeah go for it, since ppl will probably just drop content in and expect it to look good. if it's not what they want, they can just wrap children in a box

Reply:
Gonna disagree here—I don't think we should set rowGap by default.

Callout's a little different in that we set margin between children we provide (CalloutHeading, CalloutText) and tell consumers to use. But here, this could be any child the consumer provides, which we have less control or opinion over. Same reason we don't tend to set outer margins by default on most things.

Reply:
oh you're right - I didn't even check callout after reading this comment, just assumed it meant we had a gap on the actual body content for controlling spacing on multiple children but we don't